ee-9/feature-pack/src/main/resources/feature_groups/standalone-ha.xml

```diff
@@ -22,7 +22,8 @@
     <feature-group name="management-interfaces"/>
     <feature-group name="access-control"/>
     <feature-group name="health"/>
-    <feature-group name="metrics" />
+    <!-- WFLY-14108 do not forget to put it back -->
```

Comment:
All such comments were fixed in following commits after I created the new `metrics` subsystem.